specs/ops-chat-list.md

```diff
@@ -0,0 +1,62 @@
+# Create an ops-chat mailing list
+
+Create an `ops-chat` mailing list for discussions around MOC operations. The
+list will explicitly *not* be for automated alerts.
+
+## Problem Description
+
+We sometimes want to have conversations or share documentation that isn't well
+suited to a chat channel. We should create a mailing list for that purposes.
+
+## Policy
+
+We will create an `ops-chat` mailing list, the membership of which should
+probably mirror the membership of the `#coredev` slack channel. We will
+initially subscribe the ops team, but membership will not be mandatory.
+
+## Alternatives & History
+
+There is an existing `ops` mailing list, but this list is used as the target
+for automated alerts and for contacts from outside the organization. We should
+keep our discussion list separate to allow people to select which
+communications they want to receive.
+
+Our local mailing lists are hosted using [mailman][]. There has been some
+interest in selecting a hosted alternative (e.g. [Google groups][],
+[groups.io][], etc). Hosted solutions may require a monthly subscription, and a
+free solution like Google Groups may introduce advertising.
+
+There are third-party mail archiving solutions available that may provide a
+better experience than the default Mailman UI.
```

Comment:
We should probably pin down the tools we will be using as part of this PR. We already have mailman setup and supports archiving so I would vote for that.

Reply:
++ It seems like we more or less settled on a mailing list, so it would be great to sketch out more of the implementation in this spec.

Reply:
This spec assumes we are using mailman. The "alternatives and history" section is for addressing other ways of solving the problem (and possibly exploring *why* we're not using those alternatives). Sorry if that wasn't clear.